fenix/app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -515,7 +518,7 @@ fun PoweredByPocketHeader(
 
             val onClickLabel = stringResource(id = R.string.a11y_action_label_pocket_learn_more)
             Column(
-                Modifier.semantics(mergeDescendants = true) {
+                modifier = Modifier.heightIn(min = 48.dp).semantics(mergeDescendants = true) {
```

Comment:
I think you want to increase the height of the `Row` on [L501](https://github.com/mozilla-mobile/firefox-android/pull/1855/files#diff-ddfde11e28bf0cd8694e43db710d9ac05a11c0122bb5a91447e8b7f90a9b5c98L501) instead of `Column` here. That way you won't have to add the padding in the icon.